packages/kit/src/nitro.ts

```diff
@@ -82,3 +83,33 @@ export function useNitro (): Nitro {
   }
   return (nuxt as any)._nitro
 }
+
+/**
+ * Add server imports to be auto-imported by Nitro
+ */
+export function addServerImports (imports: Import[]) {
+  const nuxt = useNuxt()
+  nuxt.hook('nitro:config', (config) => {
+    config.imports = config.imports || {}
+    config.imports.autoImport = true
```

Comment:
Why this??

Reply:
don't we need to enable autoimport to make it work ?